### ISSUE TITLE
Need to Update Version http ^0.13.0 to http ^1.0.0 [Latest Version]

### ISSUE THREAD

**Issue by tusharbhambere** [Baseflow/flutter_cached_network_image#843]
## 🏗 Enhancement Proposal

Error -: google_fonts >=4.0.5 depends on http ^1.0.0, flutter_cache_manager ^3.3.0 is incompatible with google_fonts >=4.0.5.

Other Packages like google_fonts Updating http Package flutter_cached_network_image Need to Update 

### Platforms affected (mark all that apply)
- [ ✅] :iphone: iOS
- [ ✅] :robot: Android

**Comment by lreardon** [comment]
Having exact issue, hoping it gets updated soon

**Comment by avinath1998** [comment]
Yep facing this exact same issue while migrating a project to Flutter 3.10. A fix is needed asap!

**Comment by deathfist** [comment]
Facing same issue here.

**Comment by charleston10** [comment]
Is there anyone working to maintain a library?

**Comment by lucasmiranda5** [comment]
I made a fork and put the settings for dart 3


`cached_network_image: 
    git:
      url: https://github.com/lucasmiranda5/flutter_cached_network_image.git
      ref: dart3
      path: cached_network_image`

**Comment by topilski** [comment]
Same issue, any updates?

**Comment by suyie001** [comment]
any progress here?

**Comment by LAPPIT** [comment]
Got the same problem:

Because no versions of cached_network_image match >3.2.3 <4.0.0 and cached_network_image 3.2.3 depends on flutter_cache_manager ^3.3.0, cached_network_image ^3.2.3 requires flutter_cache_manager ^3.3.0.
Because flutter_cache_manager 3.3.0 depends on http ^0.13.0 and no versions of flutter_cache_manager match >3.3.0 <4.0.0, flutter_cache_manager ^3.3.0 requires http ^0.13.0.
Thus, cached_network_image ^3.2.3 requires http ^0.13.0.
So, because minsockerbitgrund depends on both cached_network_image ^3.2.3 and http ^1.0.0, version solving failed.

**Comment by madlick71** [comment]
Same problem here.

**Comment by ttb-inc** [comment]
```
dependency_overrides:
  http: 1.0.0
```

I'm using it temporarily as shown above.
Please let me know if this is a good idea.

Hopefully we can get this resolved quickly and safely.
Thanks.

**Comment by JosephNK** [comment]
Is the repository no longer live and updated?
I don't see anything activated.

**Comment by ttb-inc** [comment]
permission_handler is also a package developed by the same team.
There was an update 12 days ago.

So, I sent a mail to the development team.
Hopefully it will be resolved.

**Comment by zhouxin1233** [comment]
Facing same issue here.

**Comment by anilthapa1939** [comment]
same issue here

**Comment by prashantsolankispaceo** [comment]
I have create fork and worked well, Put in `pubspec.yaml` file as below:

  ```
  dependencies:
    cached_network_image:
      git:
        url: https://github.com/prashantsolankispaceo/cached_network_image.git
        path: cached_network_image
  ```

**Comment by nazarpunk** [comment]
Facing same issue here.

**Comment by olavurs** [comment]
Same here

**Comment by nickyrferry** [comment]
same! Any update from dev team?

**Comment by mateusz-bak** [comment]
Facing the same issue. The solution from @prashantsolankispaceo works so maybe we can see this implemented into this package? :pray:

**Comment by prashantsolankispaceo** [comment]
> Facing the same issue. The solution from @prashantsolankispaceo works so maybe we can see this implemented into this awesome package? 🙏

@mateusz-bak you can use in your live project, let me know if you find any issues, I'm also using in live application

**Comment by JosephNK** [comment]
@ttb-inc Did you get any answers from them?

**Comment by ttb-inc** [comment]
> @ttb-inc Did you get any answers from them?

This is the team's response email.

```
Hello,

Thank you for your e-mail. 
Please use github for reporting issues and asking questions. 

Kind regards
```

**Comment by SlymDev** [comment]
Also waiting for an updete on this...{-_-}
